include/solvers/mpm_base.h

```diff
@@ -156,6 +162,20 @@ class MPMBase : public MPM {
   //! \param[in] damping_props Damping properties
   bool initialise_damping(const Json& damping_props);
 
+ protected:
+  // Variable list
+  tsl::robin_map<std::string, VariableType> variables_ = {
```

Comment:
Could we move this to constructor?

Reply:
I am not sure which constructor do you mean here, @kks32

Reply:
`MPMBase` constructor

Reply:
Any thoughts why moving it to MPMBase constructor? Wouldn't it be just the same?

Reply:
Will be cleaner to add more variables in the constructor, especially since we have more than 1 variable